.github/workflows/build.yaml

```diff
@@ -0,0 +1,71 @@
+name: Main build
+
+on:
+    push:
+        branches: [main]
+
+env:
+  REGISTRY: ghcr.io
+  NAMESPACE: galasa-dev
+  IMAGE_TAG: main
+    
+jobs:
+    build-extensions:
+        name: build and Test Extensions 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+            
+            - name: Print githash
+              run: |
+                echo $GITHUB_SHA > ./extensions.githash
+            
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+  
+            - name: Test and Build Extensions using gradle
+              env:
+                GITHUB_ACTOR: ${{github.actor}}
+                GITHUB_TOKEN: ${{ github.token }}
+              run: |
+                gradle -x sign -b galasa-extensions-parent/build.gradle check publish --info \

```

Comment:
What does `-x sign` mean? I don't believe this is part of our current builds so can we remove?
Lets remove `--info` as it will crowd the log, the default setting is probably fine

Reply:
the -x ignore the sign feature since gpg signing was not present in the tekton builds for this repository

Reply:
I thought since signing was absent in the tekton builds pipeline, we won't need it, so -x flag was to skip the signing process. Making changes now.

Reply:
done